### DISCUSSION TITLE
For feature requests and bug reports, please file them in the `Issues` section

### DISCUSSION THREAD

**Discussion by CarterLi** [fastfetch-cli/fastfetch#787] [Announcements]
Threads pasted here are likely to be ignored.

**Comment by Wirelary** [comment]
This started popping up when i updated to KDE Plasma 6.6

<img width="845" height="263" alt="fastfetch, more like, idk finish the joke for me im lazy" src="https://github.com/user-attachments/assets/57c3d491-73db-4bfa-b658-51748c5fb91b" />

In the case this isn't a fastfetch issue, i will also report this to the KDE Plasma github page


config:

{
  "$schema": "https://github.com/fastfetch-cli/fastfetch/raw/dev/doc/json_schema.json",
  "logo": {
    "type": "auto",        // Logo type: auto, builtin, small, file, etc.
    "source": "fedora_small",      // Built-in logo name or file path
    "width": 7,           // Width in characters
    "height": 5,          // Height in characters
    "padding": {
        "top": 0,          // Top padding
        "left": 0,         // Left padding
        "right": 2         // Right padding
    },
    "color": {             // Override logo colors
        "1": "blue",
        "2": "blue"
    }
},

 "display": {
        "separator": " "
    },
    

  "modules": [
    {
            "type": "datetime",
            "format": "{11}-{3}-{1}",
            "key": " Today is",
            "keyColor": "green", 
        },
    
    {
    "type": "custom",
    "key": ":-|------------------|-:",
    "keyColor": "white"
    },
    
    {
        "type": "os",
        "key": " Distro:",
        "keyColor": "blue",
        "format": "{name}"
    },
    
    {
    "type": "kernel",
    "key": " Kernel:",
    "keyColor": "white",
    },
    
    {
        "type": "cpu",
        "format": "{name}",
        "key": "󰻠 CPU:",
        "keyColor": "cyan"
    },
    {
        "type": "de",
        "key": " DE:",
        "keyColor": "yellow"
    },
    {
    "type": "packages",
    "compact": true,
    "key": "󰏖 Packages:",
    "keyColor": "red"
    },
    {
    "type": "uptime",
    "key": " Uptime:",
    "keyColor": "#D2B48C",
    },
    
  ]
}
